ros/src/common/libs/diagnostics_lib/fake_autoware_nodes/CMakeLists.txt

```diff
@@ -0,0 +1,91 @@
+cmake_minimum_required(VERSION 2.8.3)
```

Comment:
please remove unnecessary comments

Reply:
I think this comment is necessary.
Another autoware nodes(ndt_maching,lidar tracker etc...) also contains this line.

Reply:
I did not said for this line. for this file.